.idea/misc.xml

```diff
@@ -25,7 +25,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
This is the line of code I was referring to... I won't reject the PR because of this, but I will expect that you spoke with the tech lead and the entire team to make them know EVERYBODY should be using the same languageLevel, I will NOT LET after this any changes back to "JDK_1_7" again